jdaviz/configs/specviz/helper.py

```diff
@@ -80,7 +80,18 @@ def get_spectral_regions(self):
         spec_regs = {}
 
         for name, reg in regions.items():
-            unit = reg.meta.get("spectral_axis_unit", u.Unit("Angstrom"))
+            try:
+                # Use the region's unit. If N/A, use the reference_data's
+                unit = reg.meta.get("spectral_axis_unit",
+                                    self.get_spectra(
+                                        self.app.get_viewer("spectrum-viewer"
+                                                            ).state.reference_data.label,
+                                        apply_slider_redshift=False
+                                        ).spectral_axis.unit
+                                    )
+            except (KeyError, LookupError, AttributeError):
+                unit = u.Unit('pixel')
```

Comment:
I think this is what the POs decided on from Slack discussions.

```suggestion
                unit = u.dimensionless_unscaled
```

I still recommend adding tests to ensure this does not result in undesirable side effects.

Reply:
Agreed. I pushed in the latest commit!